### PR TITLE
remove some unused functions

### PR DESCRIPTION
**Pull request by skimo-openhub**
Most of these were related to PPCG, but some have never been used at all.

### CODE REVIEW

**Comment by prigoyal** [comment]
this looks good to me, thanks very much @skimo-openhub :) merging this PR now, it looks harmless and no one has looked at it for the whole day. Please let me know if you rather prefer to wait for tagged reviewers in future.

**Comment by skimo-openhub** [comment]
On Mon, Feb 26, 2018 at 07:26:14PM -0800, Priya Goyal wrote:
> this looks good to me, thanks very much @skimo-openhub :) merging this PR now, it looks harmless and no one has looked at it for the whole day. Please let me know if you rather prefer to wait for tagged reviewers in future. 

I would indeed prefer commits to get reviewed before they get merged and
not just my own commits.

skimo

**Comment by ftynse** [comment]
Did we disable the github policy requesting at least one approve before merging?..

**Comment by ftynse** [comment]
> no one has looked at it for the whole day

May I remind you that people work from different time zones and often have other obligations than reacting at every single ping asap ?

File: include/tc/external/detail/islpp-inl.h [hunk 1]
```diff
@@ -218,97 +218,4 @@ inline isl::set operator&(isl::point P1, isl::set S2) {
...
-  std::vector<long> vals;
-  std::vector<const char*> names;
-  for (const auto& kvp : nameMap) {
-    names.push_back(kvp.first.c_str());
-    vals.push_back(kvp.second);
-  }
-  return makePoint(s, names, vals);
-}
-
-inline long evalIntegerAt(isl::aff a, isl::point pt) {
```
Comment (ftynse) [inline include/tc/external/detail/islpp-inl.h]: I suppose this can be replaced by the recently introduced isl function?

**Comment by skimo-openhub** [comment]
On Tue, Feb 27, 2018 at 10:25:06AM +0000, ftynse wrote:
> Did we disable the github policy requesting at least one approve before merging?..

Apparently; merging was enable for me as ssone as the tests finished.

skimo

**Comment by skimo-openhub** [comment]
On Tue, Feb 27, 2018 at 10:27:48AM +0000, ftynse wrote:
> ftynse commented on this pull request.
> -inline isl::point makePoint(
> -    isl::space s,
> -    std::unordered_map<std::string, long> nameMap) {
> -  std::vector<long> vals;
> -  std::vector<const char*> names;
> -  for (const auto& kvp : nameMap) {
> -    names.push_back(kvp.first.c_str());
> -    vals.push_back(kvp.second);
> -  }
> -  return makePoint(s, names, vals);
> -}
> -
> -inline long evalIntegerAt(isl::aff a, isl::point pt) {
> 
> I suppose this can be replaced by the recently introduced isl function?

Yes, see commit message.

skimo

**Comment by prigoyal** [comment]
>> May I remind you that people work from different time zones and often have other obligations than reacting at every single ping asap ?

agreed and like I said, I reviewed it and it looked fine to me. So I merged it. I am not pushing anyone to look at any PR, feel free to do it at your pace.

**Pull request merged**